deploy/test/index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head></head><body><h1>Hello World!</h1></body></html>
```

Comment:
UPDATE: doing so breaks the integration test, as the file isn't supposed to have a newline at the end.